libraries/AC_PrecLand/AC_PrecLand.h

```diff
@@ -116,24 +154,35 @@ class AC_PrecLand
     AP_Float                    _land_ofs_cm_y;     // Desired landing position of the camera right of the target in vehicle body frame
     AP_Float                    _accel_noise;       // accelerometer process noise
     AP_Vector3f                 _cam_offset;        // Position of the camera relative to the CG
+    AP_Int8                     _strict;            // PrecLand strictness
+    AP_Int8                     _max_retry_num;     // PrecLand Maximum number of retires to a failed landing
+    AP_Float                    _retry_timeout_sec; // Time for which vehicle continues descend even if target is lost. After this time period, vehicle will attemp a landing retry depending on PLND_STRICT param.
+    AP_Int8                     _retry_behave;      // Action to do when trying a landing retry
+    AP_Float                    _sensor_min_alt;     // PrecLand minimum height required for detecting target
+    AP_Float                    _sensor_max_alt;     // PrecLand maximum height the sensor can detect target
 
     uint32_t                    _last_update_ms;    // system time in millisecond when update was last called
     bool                        _target_acquired;   // true if target has been seen recently after estimator is initialized
     bool                        _estimator_initialized; // true if estimator has been initialized after few seconds of the target being detected by sensor
     uint32_t                    _estimator_init_ms; // system time in millisecond when EKF was init
     uint32_t                    _last_backend_los_meas_ms;  // system time target was last seen
+    uint32_t                    _last_precland_output_ms;   // last time PrecLand library had a output of the landing target position
 
     PosVelEKF                   _ekf_x, _ekf_y;     // Kalman Filter for x and y axis
     uint32_t                    _outlier_reject_count;  // mini-EKF's outlier counter (3 consecutive outliers lead to EKF accepting updates)
-    
+
     Vector3f                    _target_pos_rel_meas_NED; // target's relative position as 3D vector
 
+    Vector3f                    _last_target_loc_rel_origin_NED;  // stores the last known location of the target horizontally, and the height of the vehicle where it detected this target in meters NED
```

Comment:
sorry for repeating myself but just to be sure, let's replace the "loc" with "pos" in this member's name.